src/Sql/NotificationCenter/dbo/Tables/Notification.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE [dbo].[Notification]
+(
+    [Id] UNIQUEIDENTIFIER NOT NULL,
+    [Priority] TINYINT NOT NULL,
+    [Global] BIT NOT NULL,
+    [ClientType] TINYINT NOT NULL,
+    [UserId] UNIQUEIDENTIFIER NULL,
+    [OrganizationId] UNIQUEIDENTIFIER NULL,
+    [Title] NVARCHAR (MAX) NULL,
+    [Body] NVARCHAR (MAX) NULL,
+    [CreationDate] DATETIME2 (7) NOT NULL,
+    [RevisionDate] DATETIME2 (7) NOT NULL,
+    CONSTRAINT [PK_Notification] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_Notification_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]) ON DELETE CASCADE,
```

Comment:
⚠️ Can get rid of all these (evil) `CASCADE`s too.

Reply:
Fixed in 006f1b4